Sources/MongoSwiftSync/ClientSession.swift

```diff
@@ -96,4 +96,45 @@ public final class ClientSession {
     public func advanceOperationTime(to operationTime: Timestamp) {
         self.asyncSession.advanceOperationTime(to: operationTime)
     }
+
+    /**
```

Comment:
ditto comment in other PR about adding `SeeAlso` link to official docs.

Reply:
Done.